change/@fluentui-react-tree-bd13a9b3-9413-43ea-91f2-9b58288a77d8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31766&commitId=51aa1cc38fe067b84a4f4d3a939ee36fbcb1e35a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default - High Contrast.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>